.azure/test-linux.yml

```diff
@@ -71,6 +71,8 @@ jobs:
             set -e
             source test-job/bin/activate
             pip install -U -r requirements.txt -r requirements-dev.txt -c constraints.txt
+            # Install setuptools-rust for building sdist
+            pip install -U -c constraints.txt setuptools-rust
             python setup.py sdist
```

Comment:
At some point we might want to replace this sort of thing with `pip install build; python -mbuild --sdist` to get full PEP-517-compatible sdist building (that would automatically isolate and install `setuptools-rust` from `pyproject.toml`), but it doesn't need to / shouldn't be in this PR.